### PR TITLE
Update x.php

### PR DESCRIPTION
**Pull request by newkhaled**

### CODE REVIEW

File: x.php [hunk 1]
```diff
@@ -23,6 +23,16 @@ public function up()
             $table->bigInteger('amount')->nullable(false);
             $table->text('details')->nullable(false); 
         });
+          Schema::create('transactions', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('transaction_type_id')->nullable(false)->references('id')->on('customers');
+            $table->foreignId('customer_id')->nullable(true)->references('id')->on('customers');
+            $table->foreignId('project_id')->nullable(true)->references('id')->on('projects');
```
Comment (khaledfathi) [inline x.php:30]: - hfdgh
- bhdg
- jhfjgf

File: x.php [hunk 2]
```diff
@@ -23,6 +23,16 @@ public function up()
             $table->bigInteger('amount')->nullable(false);
             $table->text('details')->nullable(false); 
         });
+          Schema::create('transactions', function (Blueprint $table) {
```
Comment (khaledfathi) [inline x.php:26]: dfgh

**Review by khaledfathi — CHANGES_REQUESTED** [review]

File: x.php (hunk 1 above)
Comment (newkhaled) [reply x.php:30]: rersasdk

File: x.php [hunk 3]
```diff
@@ -23,6 +23,16 @@ public function up()
...
+            $table->id();
+            $table->foreignId('transaction_type_id')->nullable(false)->references('id')->on('customers');
+            $table->foreignId('customer_id')->nullable(true)->references('id')->on('customers');
+            $table->foreignId('project_id')->nullable(true)->references('id')->on('projects');
+            $table->date('date')->nullable(false);
+            $table->time('time')->nullable(false);
+            $table->bigInteger('amount')->nullable(false);
```
Comment (newkhaled) [inline x.php:33]: gfd

File: x.php (hunk 1 above)
Comment (newkhaled) [reply x.php:30]: ghhdhdf

File: x.php (hunk 2 above)
Comment (newkhaled) [reply x.php:26]: egsdfgfd

File: x.php (hunk 3 above)
Comment (newkhaled) [reply x.php:33]: ghgfdhhd

**Review by khaledfathi — APPROVED** [review]

**Pull request merged**